lib/lists/sites-list.js

```diff
@@ -33,15 +33,30 @@ function constructMediaPath(site) {
  * @returns {Array}
  */
 function generateSitesIndexOps(acc, site) {
-  var mediaDir = path.resolve(process.cwd(), site.assetDir, 'media', 'sites', site.slug),
-    mediaDirFiles = files.getFiles(mediaDir),
+  var siteId = site.subsiteSlug || site.slug,
```

Comment:
Let's declare these properties as `let`
```suggestion
  let siteId = site.subsiteSlug || site.slug,
```